### ISSUE TITLE
Is the space fixed?

### ISSUE THREAD

**Issue by Open2logic** [joomla/joomla-cms#15386]
### Steps to reproduce the issue
In the administrator.
Force a Warning and Mesasge notifications
![screen shot 2017-04-18 at 15 52 32](https://issues.joomla.org/uploads/1/ea193e15dd764ed5a3b5a5425f523fac.jpg)

### Closed the Message, all go up

![screen shot 2017-04-18 at 15 55 54](https://issues.joomla.org/uploads/1/963afb57fbe326da2c16f302bcb98767.jpg)
### 
### Closed the Warning, the zone is fixed
![screen shot 2017-04-18 at 15 56 06](https://issues.joomla.org/uploads/1/5065a9807ba3c55760c2b8d2a35e8b15.jpg)
### Expected result
All go up

### Actual result
The Warning area is still fixed , after closing the message

### System information (as much as possible)
Joomla 3.7 Rc2

**Comment by brianteeman** [comment]
Please explain a real world case on how to get both those messages at the same time

**Comment by C-Lodder** [comment]
Can't replicate this at all.

@brianteeman - Go to article manage, select some articles and unpublish them. Or you can inject the following HTML via your inspector, directly above the main `<form>`

```
<div id="system-message-container" class="j-toggle-main span10">
	<div class="alert alert-success">
		<button type="button" class="close" data-dismiss="alert">×</button>
		<h4 class="alert-heading">Message</h4>
		<div class="alert-message">2 articles published.</div>
	</div>

	<div class="alert alert-warning">
		<button type="button" class="close" data-dismiss="alert">×</button>
		<h4 class="alert-heading">Warming</h4>
		<div class="alert-message">oops, something went wrong</div>
	</div>
</div>
```

**Comment by dgt41** [comment]
@C-Lodder I think this line might be a problem for some browsers
https://github.com/joomla/joomla-cms/blob/staging/media/system/js/core-uncompressed.js#L344
I propose to change it to:
messageContainer.offsetHeight = 0;

**Comment by C-Lodder** [comment]
Why are we using `el.offsetHeight` in the first place?

**Comment by Open2logic** [comment]
Only happens in Chrome.
Create an admin menu and one buttom,
The module for this menu, In the options: Check Menu in YES.
Now the admin show a message: 
"Warning
The administrator menu Admin does not contain - Menu Manager, Module Manager, Components Container. Click to turn on the menu recovery mode."


Closed this message and the zone is fixed.
Ok for Explorer And Firefox<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15386">issues.joomla.org/tracker/joomla-cms/15386</a>.</sub>

**Comment by C-Lodder** [comment]
Not sure why considering the BS2 docs work perfectly fine.

**Comment by dgt41** [comment]
@C-Lodder I don't have access to a 3.x install right now, but the problem is in `Joomla.removeMessages`.
Maybe `messageContainer.offsetHeight = '';` or resetting the margin(s) ?

**Comment by C-Lodder** [comment]
I'll take a look when I'm home. Have a hunch is something to do with the rubbish in `template.js`

**Comment by dgt41** [comment]
check also the initial commit: https://github.com/joomla/joomla-cms/pull/5257

**Comment by C-Lodder** [comment]
My hunch was correct. `template.js`

**Comment by C-Lodder** [comment]
It's all to do with that crappy sidebar that can slide in/out, which then changes some classes to manipulate the width. Enjoy fixing.

**Comment by dgt41** [comment]
WON'T.
Let's introduce custom elements, alerts seems an easy starting point

**Comment by C-Lodder** [comment]
Custom elements in 3.x?

**Comment by dgt41** [comment]
Oops, thought it was for 4...

**Comment by C-Lodder** [comment]
J4 we have fixed alerts ;)

**Comment by dgt41** [comment]
In J4 we need to go away from bootscrap... 😀

**Comment by brianteeman** [comment]
Closed as a duplicate of #12155
